templates/projects/task_presenter_editor.html

```diff
@@ -282,6 +322,11 @@ <h4 class="modal-title"><strong>{{project.name}}</strong>: {{ _('Task Presenter
     }
   }
 });
+
+setInterval(function() {
+  editor.refresh();
```

Comment:
What does this do? Why is it needed?

Reply:
The code mirror not working properly but Ill use the event when clicking the tab.